lib/galaxy/datatypes/binary.py

```diff
@@ -1756,6 +1756,37 @@ def sniff(self, filename):
             return False
 
 
+class DAA(Binary):
+    """
+    Class describing an DAA (diamond alignment archive) file
+    >>> from galaxy.datatypes.sniff import get_test_fname
+    >>> fname = get_test_fname( 'diamond.daa' )
+    >>> DAA().sniff( fname )
+    True
+    >>> fname = get_test_fname( 'interval.interval' )
+    >>> DMND().sniff( fname )
+    False
+    """
+    file_ext = "dmnd"
+    edam_format = ""
+
+    def __init__(self, **kwd):
+        Binary.__init__(self, **kwd)
+                                          
+        self._magic = binascii.unhexlify("6be33e6d47530e3c")
+
+    def sniff(self, filename):
+        # The first 8 bytes of any daa file are 0x3c0e53476d3ee36b
+
+        try:
+            header = open(filename, 'rb').read(8)
+            if header == self._magic:
+                return True
+            return False
+        except Exception:
```

Comment:
I don't think this needs to be wrapped in try/except, the framework will treat an Exception as False anyway. It is probably better to use a `with` statement for  opening/closing the file though.

Reply:
OK. Also changed this in the DMND datatype from where I copy-pasted the code